components/range-control/index.js

```diff
@@ -26,7 +26,7 @@ function RangeControl( {
 } ) {
 	const id = `inspector-range-control-${ instanceId }`;
 	const onChangeValue = ( event ) => onChange( Number( event.target.value ) );
-	const initialSliderValue = value || initialPosition || '';
+	const initialSliderValue = value === undefined ? initialPosition || '' : value;
```

Comment:
Why would an empty string ever be provided as a value to the range?

Reply:
Hi @aduth,

> Why would an empty string ever be provided as a value to the range?

If we provide undefined, and after we set a value we get an error saying the component is passing from uncontrolled to controlled input. If we set 0 we are setting a value when a value may not be set. That's the reason an empty string was used.

Reply:
>If we provide undefined, and after we set a value we get an error saying the component is passing from uncontrolled to controlled input.

But isn't it a legitimate error for the developer to be made aware of to be changing between uncontrolled and controlled?

Reply:
There are many cases where we are passing a value but at the start, that value is undefined / unknown. This would force all the users of the component with an undefined starting value in the variable to handle that case and pass another thing when the value is undefined like the empty string to avoid this errors.

Reply:
Ok, I didn't realize React had special handling of the empty string as a value. It seemed out of place, particularly for the numeric range field. In reflection, appears fine / expected.